lib/open_api_spex/plug/swagger_ui.ex

```diff
@@ -90,11 +90,19 @@ defmodule OpenApiSpex.Plug.SwaggerUI do
           request.headers["x-csrf-token"] = "<%= csrf_token %>";
           return request;
```

Comment:
@mbuhot The following change in the interceptor fixed CORS issues in my environment. Essentially, part of CORS involves a negotiation of what headers are allowed to cross domain boundaries. The CSRF token is not one of those headers, so we need to make sure it's not shoved into the intercept when talking to another server.

```suggestion
          server_base = window.location.protocol + "//" + window.location.host
          if(request.url.startsWith(server_base)) {
            request.headers["x-csrf-token"] = "<%= csrf_token %>";
          } else {
            delete request.headers["x-csrf-token"]
            return request;
          }
```